.gitignore

```diff
@@ -9,3 +9,6 @@ compile_commands.json
 .vscode/
 *.log
 *.csv
+
+# Generated flatbuffer header for simulation device
+device/simulation/tt_simulation_device_generated.h
```

Comment:
Normally all generated files (binary and text) live in the build directory.  Is there something that forces us to generate a file into our source tree?

Reply:
I just have it generating into the source tree right now, but it should definitely go into build directory now that I think of it. Will change.

Not sure if this will affect anything else, as I will now have to `target_include_directories` for `${CMAKE_BINARY_DIR}`? And it seems like it's installing all headers into a certain place?

Reply:
That's correct, we (and if the header is public, then others, too) will need to include the destination inside the bindir in the search path.  This is normal and expected.  And one of several reasons why we should be cognizant of which headers are public vs. which are private.

Reply:
Will be private, waiting for https://github.com/tenstorrent/tt-umd/pull/224